packages/hardhat-core/src/internal/cli/project-creation.ts

```diff
@@ -42,11 +42,11 @@ const HARDHAT_PACKAGE_NAME = "hardhat";
 const PROJECT_DEPENDENCIES: Dependencies = {};
 
 const ETHERS_PROJECT_DEPENDENCIES: Dependencies = {
-  "@nomicfoundation/hardhat-toolbox": "^4.0.0",
+  "@nomicfoundation/hardhat-toolbox": "^5.0.0",
 };
 
 const VIEM_PROJECT_DEPENDENCIES: Dependencies = {
```

Comment:
We should add `@nomicfoundation/hardhat-ignition` to `PEER_DEPENDENCIES` to ensure it gets installed, as not all package managers install peer dependencies by default. If there are other new peer dependencies, we should also add them to the appropriate objects.